File: muridku/muridku.backend/GlobalDbProcessor/Muridku.QueryRequestReceiver/Models/Dbs/Member.cs
using System;

namespace Muridku.QueryRequestReceiver.Models.Dbs
{
  public class Member : ModelDbBase
  {
    public string name { get; set; }
    public string address { get; set; }
    public string gender { get; set; }
    public DateTime? birth_dt { get; set; }
    public string birth_place { get; set; }
    public string mobile_phn { get; set; }
    public string inst_type { get; set; }
    public long city_id { get; set; }
    public long? institution_id { get; set; }
    public long? faculty_id { get; set; }
  }
}

File: muridku/muridku.web/app/Models/Member.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Notifications\Notifiable;

class Member extends Model
{
    use HasFactory, Notifiable;

    protected $table = 'member';
    const CREATED_AT = 'dtm_crt';
    const UPDATED_AT = 'dtm_upd';

    protected $fillable = [
        'name',
        'address',
        'gender',
        'birth_dt',
        'birth_place',
        'mobile_phn',
        'inst_type',
    ];

    protected $hidden = [
        'city_id',
        'institution_id',
        'faculty_id',
        'dtm_crt',
        'usr_crt',
        'dtm_upd',
        'usr_upd'
    ];
}
